src/material/sidenav/sidenav.md

```diff
@@ -209,6 +209,11 @@ Similarly, the `<mat-sidenav-content>` should be given a role based on what it c
 represents the primary content of the page, it may make sense to mark it `role="main"`. If no more
 specific role makes sense, `role="region"` is again a good fallback.
 
+#### Focus management
+The sidenav captures focus automatically by default, unless it's in the side mode. Focus capturing can be enabled or disabled explicitly, using the `autoFocus` input.
```

Comment:
How do you feel about this? I think it makes it just a little bit easier to follow

> The sidenav has the ability to capture focus automatically. This behavior is turned on by default for the `push` and `over` mode sidenavs, while it is off by default for the `side` mode sidenav. You can set it explicitly regardless of the mode using the `autoFocus` input.

Reply:
I think, the older version is a bit laconic. Also it only talks about autoFocus property, without adding knowledge about sidenav mode definitions. Said so, you're version is better in flow. The sentences flow one after another smoothly.
In conclusion, I would still leave my version :)

Reply:
In particular, I don't like the phrase "automatically by default". I understand what you're trying to say, but its confusing because people often use the "automatically" to mean "by default".

> only talks about autoFocus property, without adding knowledge about sidenav mode definitions

I think we should mention the other modes explicitly, its easier to understand when you tell people what modes _do_ have this behavior rather than (or in addition to) which ones don't.

Reply:
> The sidenav has the ability to capture focus. This behavior is turned on for `push `and `over` modes and it is off for `side` mode. you can change it's default behavior by `autoFocus `input

what do you think about that?

Reply:
I like that. It's succinct and addresses the concerns I had